sdk/typescript/src/signers/txn-data-serializers/local-txn-data-serializer.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Base64DataBuffer } from '../../serialization/base64';
+import {
+  CallArg,
+  MoveCallTx,
+  registerTypes,
+  SuiAddress,
+  TransactionData,
+  TypeTag,
+} from '../../types';
+import { bcs as BCS } from '@mysten/bcs';
+import {
+  MoveCallTransaction,
+  MergeCoinTransaction,
+  SplitCoinTransaction,
+  TransferObjectTransaction,
+  TransferSuiTransaction,
+  PublishTransaction,
+  TxnDataSerializer,
+} from './txn-data-serializer';
+import { Provider } from '../../providers/provider';
+
+// For commonjs scenario need to figure out how to do it differently
+const bcs = registerTypes(BCS);
```

Comment:
I'm a bit confused why you're constructing it  like this (with the `registerTypes` function that runs initializers) vs just registering directly in that module itself.

Reply:
Good point! Just moved it back to sui-bcs.ts